staging/src/k8s.io/client-go/pkg/api/helpers.go

```diff
@@ -465,6 +465,44 @@ const (
 	ObjectTTLAnnotationKey string = "node.alpha.kubernetes.io/ttl"
 )
 
+// AddOrUpdateTolerationInPod tries to add a toleration to the pod's toleration list.
+// Returns true if something was updated, false otherwise.
+func AddOrUpdateTolerationInPod(pod *Pod, toleration *Toleration) (bool, error) {
```

Comment:
I think we will reject pods if failed to update it, e.g. `err != nil`; but when will this func return error (`err != nil`)? My understanding is that the following will be the return value of this func, although it always return err(nil) for now:
* if add or update, return `true, nil`
* if no action (existed), return `false, nil`
* if any error, return `_, err`

Current behavior seems correct, but confused by its return value, e.g. when will this func return error? :).